### PR TITLE
Add docs for usage

### PR DESCRIPTION
**Pull request by sbellem**

### CODE REVIEW

File: docs/usage.rst [hunk 1]
```diff
@@ -83,13 +95,13 @@ and then transfers it to Bob:
```
Comment (sohkai) [inline docs/usage.rst]: Should be `creation_tx = bdb.transactions.retrieve(creation_tx['id'])`

File: docs/usage.rst [hunk 2]
```diff
@@ -113,12 +125,49 @@ Bob is the new owner:
...
+
+.. code-block:: python
+
+    import logging
+
+    from bigchaindb_driver import BigchainDB
+    from bigchaindb_driver.exceptions import NotFoundError
+
+    logger = logging.getLogger(__name__)
+    logging.basicConfig(format='%(asctime)-15s %(status)-3s %(message)s')
```
Comment (sohkai) [inline docs/usage.rst]: Maybe just use `print` here to make it less complicated? If they want to use the logger, I'm sure they'll be able to figure out how elsewhere.

Comment (sbellem) [inline docs/usage.rst]: :smile: This was really intentional!

Comment (sohkai) [inline docs/usage.rst]: I could tell ;).

**Comment by sohkai** [comment]
LGTM; we can move the error handling to its own section later.

**Pull request merged**